js/main.js

```diff
@@ -1 +1,50 @@
-/* Javascript goes here! */
+$(function(){
+  console.log("Listo!");
+  requestWeatherData();
+    $("form").submit(function(event){
+    event.preventDefault();
+    document.querySelector('.city-error').style.display = "none";
+    $('.icon img').remove();
+    requestWeatherData();
+    })
+})
+
+function requestWeatherData(){
+  console.log("requesting your data!");
+    $.ajax({
+      method: 'GET',
+      url: 'http://api.openweathermap.org/data/2.5/weather',
+      data:$('form').serialize(),
+      dataType: 'json',
+      success: onSuccess,
+      error: onError
+    })
+}
+
+function onSuccess(data){
```

Comment:
You'll want to make sure to clear out the text input after a successful data display too!